app/pb_kits/playbook/pb_pill/_pill.scss

```diff
@@ -15,7 +15,10 @@ $pb_pill_height: 22px;
 
   @each $color_name, $color_value in $status_color_text {
     &[class*=_#{$color_name}]  {
-      background: rgba($color_value, $opacity-1);
+        background: rgba($color_value, $opacity-1);
+        &.dark {
+          background: rgba(mix($bg_dark, $color_value, 10%), $opacity_5);
+        }
```

Comment:
Let's handle this in an upcoming story

```suggestion
```

Reply:
@kellyeryan with Github suggestions like @jasoncypret has here you can simply choose “commit suggestion” to update your PR.